src/components/common/FileUploadPreview.vue

```diff
@@ -83,29 +83,38 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
   //  is using a different validation approach or updating all components to use vuelidate.  Have decided
   //  to do this for the time being.
   private async validateFileInput (file: File): Promise<boolean> {
-    this.customErrorMessages = []
+    // clear any existing error messages
+    this.setCustomErrorMessage(null)
```

Comment:
The old code set the custom error message here and via a watcher. However, the watcher only fires if the new prop value is different than the previous one, and that caused errors to not display if, for example, after selecting a duplicate file, you then selected an invalid one.

Now, there is still a watcher on the prop, but also this way to update the local variable.